src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.html

```diff
@@ -1,6 +1,7 @@
 <button type="button" class="btn btn-sm btn--capture-keystroke"
             [ngClass]="{'btn-default': !record, 'btn-info': record}"
-            (click)="start()"
+            (keydown.enter)="onButtonEnterDown($event)"
```

Comment:
With this the pressing of the enter could be intercepted at the button level, so the component would only get the click when capturing is in progress.